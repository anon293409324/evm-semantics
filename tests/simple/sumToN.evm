account:
-   id: 2
-   balance: 40
-   program: PUSH 0 ; PUSH 1 ; MSTORE ; PUSH 0 ; MLOAD ; PUSH 10 ; JUMP1 ; PUSH 1 ; PUSH 1 ; RETURN ; PUSH 1 ; MLOAD ; PUSH 0 ; MLOAD ; ADD ; PUSH 1 ; MSTORE ; PUSH 1 ; PUSH 0 ; MLOAD ; SUB ; PUSH 0 ; MSTORE ; PUSH 3 ; JUMP
-   storage: 

account:
-   id: 1
-   balance: 40
-   program: PUSH 5 ; PUSH 0 ; MSTORE ; PUSH 1 ; PUSH 1 ;  
             PUSH 1 ; PUSH 0 ; PUSH 4000 ; PUSH 2 ; CALL
-   storage:
.Accounts


transaction:
-   to: 1
-   from: .AcctID
-   data:
-   value: 0
-   gasPrice: 1
-   gasLimit: 10000

.Transactions
