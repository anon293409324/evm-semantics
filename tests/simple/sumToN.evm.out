<T> <k> . </k> <id> 1 </id> <pc> 50 </pc> <gas> 9607 </gas> <wordStack> .WordStack </wordStack> <localMem> 0 |-> 0 1 |-> 15 </localMem> <gasPrice> 1 </gasPrice> <callStack> .CallStack </callStack> <accounts> <account> <acctID> 1 </acctID> <program> 0 |-> ( PUSH 5 ) 1 |-> ( PUSH 0 ) 2 |-> MSTORE 3 |-> ( PUSH 0 ) 4 |-> ( PUSH 1 ) 5 |-> MSTORE 6 |-> ( PUSH 0 ) 7 |-> MLOAD 8 |-> ( PUSH 12 ) 9 |-> JUMP1 10 |-> ( PUSH 50 ) 11 |-> JUMP 12 |-> ( PUSH 1 ) 13 |-> MLOAD 14 |-> ( PUSH 0 ) 15 |-> MLOAD 16 |-> ADD 17 |-> ( PUSH 1 ) 18 |-> MSTORE 19 |-> ( PUSH 1 ) 20 |-> ( PUSH 0 ) 21 |-> MLOAD 22 |-> SUB 23 |-> ( PUSH 0 ) 24 |-> MSTORE 25 |-> ( PUSH 6 ) 26 |-> JUMP </program> <storage> 0 |-> 30 1 |-> 20 2 |-> 25 </storage> <balance> 40 </balance> </account> </accounts> </T>
